### ISSUE TITLE
Same world every time

### ISSUE THREAD

**Issue by DrNeonDinosaur** [Kasax007/Modpack-Development#10]
Every time I make a new world its the same savannah gravel covered island with a ravine off the shore of a graven savannah biome. I literally cant get any other world no matter the seed I use, world settings, anything.

**Comment by Kasax007** [comment]
What version are you playing on?

**Comment by Kasax007** [comment]
So the V5 version I assume? 
This is interesting because I always get different worlds, but I've gotten multiple reports of people only getting the same world. Hope I can find the issue that causes this.

**Comment by DrNeonDinosaur** [comment]
its not the exact same. sometimes its more flat (when I made it an amplified world it was more hilly but the same still and the random things like pirate ships change but its always the same island with a ravine next to a big savannah. I was hoping there was a mod I could disable that would fix this or something. I tried reinstalling and it didn't fix it and its still the same world after uninstalling and reinstalling which I thought was weird.
